src/js/common/schemaform/state/selectors.js

```diff
@@ -1,3 +1,4 @@
 export const getFormData = state => state.form.data;
 export const getFormPages = state => state.form.pages;
 export const getSubmission = state => state.form.submission;
+export const getReviewPageOpenChapters = state => state.form.reviewPageView.openChapters;
```

Comment:
Not happy with this being on `state.form`- conceptually it's not related to form data and might be better at the on the root or in something like `state.viewState`. We don't have a lot of viewState reducers so I dunno if it's worth it to create one just for this.